code/modules/mob/living/carbon/carbon.dm

```diff
@@ -269,14 +269,22 @@ mob/living
 						)
 			// BEGIN HUGCODE - N3X
 			else
-				if (istype(src,/mob/living/carbon/human) && src:w_uniform)
-					var/mob/living/carbon/human/H = src
-					H.w_uniform.add_fingerprint(M)
-				playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
-				M.visible_message( \
+				if(M.zone_sel.selecting == "head")
+					playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+					M.visible_message( \
+					"\blue [M] pats [src] on the head.", \
```

Comment:
Color macros for text are bad practice, use `"<span class='notice'>[M] pats [src] on the head.</span>"` instead. Same for the line below.